docs/docs/smart-contracts/ownable.md

```diff
@@ -82,9 +83,11 @@ pub mod my_ownable {
         contracts::ownable::*,
         modifiers,
     };
+    use ink_storage::traits::SpreadAllocate;
+
 
    #[ink(storage)]
-   #[derive(Default, OwnableStorage)]
+   #[derive(Default, SpreadAllocate, OwnableStorage)]
```

Comment:
Better to call `ink_lang::codegen::initialize_contract` in any constructor, so please add it everywhere=)